### ISSUE TITLE
Enhance input help

### ISSUE THREAD

**Issue by Emil-18** [nvaccess/nvda#15891]
### Is your feature request related to a problem? Please describe.
No
### Describe the solution you'd like
As input help works now, when pressing for example shift plus slash in the english keyboard layout, NVDA reports shift+slash instead of question. I think NVDA instead should report the actual character if the key combination has no named script attached to it. In this way, the user can check what character they will type when pressing different key combinations. I also think NVDA should have a quick way to report to the user if the gesture has a named script attached to it, for example replacing the plus sign between keys with space  if the gesture isn't asosiated with a script. In this way, when learning NVDA commands, the user can quickly check if a gesture has an asosiated script, and if not, move quickly to the next key combination
### Additional context
Narrator, JAWS, and Voiceover all have this functionality.

**Comment by CyrilleB79** [comment]
> As input help works now, when pressing for example shift plus slash in the english keyboard layout, NVDA reports shift+slash instead of question. I think NVDA instead should report the actual character if the key combination has no named script attached to it. In this way, the user can check what character they will type when pressing different key combinations.

Duplicate of #6621

> > I also think NVDA should have a quick way to report to the user if the gesture has a named script attached to it, for example replacing the plus sign between keys with space if the gesture isn't asosiated with a script. In this way, when learning NVDA commands, the user can quickly check if a gesture has an asosiated script, and if not, move quickly to the next key combination

I do not understand this part. Can you be more specific?

**Comment by Emil-18** [comment]
@CyrilleB79 I find it useful when it doesn't say plus between key names when the key combination isn't a script, because then I know immediately that it doesn't have a script asosiated with it.

**Comment by XLTechie** [comment]
> Note that in this case, Jaws also reports "control alt R" for JawsKey+control+alt+R, probably because the JawsKey is never passed to the system.

I'm not sure any conclusions can be drawn based on that.

It is likely either keyboard or BIOS dependent. I have certain combinations 
which report in full while using the built-in keyboard as opposed to a USB 
keyboard, and vice-versa. Insert and Capslock were never intended as true 
modifier keys by keyboard manufacturers, I believe, and the combination with 
other keys is implemented haphazardly.

**Comment by seanbudd** [comment]
Could you rewrite the issue to only talk about your second problem, not the duplicate?

**Comment by CyrilleB79** [comment]
@Emil-18, thanks for having rewritten the issue.

Could you please also change the title to something more specific? Thanks.

**Comment by seanbudd** [comment]
Closing in favour of #6621
